app/views/staff/events/show.html.haml

```diff
@@ -103,24 +117,26 @@
               %tr
                 %td.text-primary
                   %strong CFP Closes Date:
-                - if event.closes_at.present?
+                - if event.closes_at && (event.closes_at > Time.current)
```

Comment:
OK, since you're using Time.current instead of Time.now, the app's timezone setting will be respected. And since the event model uses Time.current, the comparison here is apples-to-apples.

It does look like timezone is being used inconsistently between different parts of the app though. I'm inclined to make a separate card for that and not hold this PR up.